### ISSUE TITLE
Reproduce MobileNetV[12] mAP Coco traning from scratch

### ISSUE THREAD

**Issue by BraginIvan** [tensorflow/models#7561]
### System information
- **What is the top-level directory of the model you are using**:
research/object_detection/models
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Linux Ubuntu 18.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.14
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**: 10.0/7.6.3
- **GPU model and memory**: RTX 2080TI 11G
- **Exact command to reproduce**: 
python3 model_main.py --pipeline_config_path=mobilenetv2/ssd_mobilenet_v2_coco.config --model_dir=mobilenetv2/model

### Describe the problem (Feature request)
I develop my own SSD architecture based ob EfficientNet-TPU-S but in parallel want to be sure that my environment setup is correct.
To be sure in my setup I have been trying to reproduce mAP of a model from [zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md). 
Steps that I done to reproduce:
- prepared minival validation set using [these ids
](https://github.com/tensorflow/models/blob/master/research/object_detection/data/mscoco_minival_ids.txt)
- check if checkpoint from zoo gives me the same mAP as in the zoo table MobileNetV1 (0.21) MobileNetV2 (0.22)
- change config https://github.com/tensorflow/models/blob/master/research/object_detection/samples/configs/ssd_mobilenet_v1_coco.config#L156. Just remove lines 156, 157 and 160 so it should be trained from scratch indefinitely
- After 1.1M steps MobileNetV1 mAP <0.16  and it is not changed for 200K steps (from 900K)
- After 550K steps MobileNetV2 mAP 0.14 and it is about the same that  MobileNetV1 has at this step.

At the same time  EfficientNet-TPU-S gives me mAP 0.2 but I don't know if  EfficientNet-TPU-S is better than  MobileNetV1 because it gives bigger mAP (0.2 > 0.16) or it worse based on mAP from documentation.
Would be great to have config that reproduces at least one model from zoo.
Have one more idea, may be I have to train from pre-trained backbone on ImageNet.

### Source code / logs
MobilenetV1
![Screenshot from 2019-09-16 09-52-27](https://user-images.githubusercontent.com/13066049/64938809-bf0c5b80-d867-11e9-9394-7a9c478a3417.png)
MobilenetV2
![Screenshot from 2019-09-16 09-55-06](https://user-images.githubusercontent.com/13066049/64938933-190d2100-d868-11e9-8c12-597ac7f9bffb.png)

**Comment by BraginIvan** [comment]
Found https://github.com/tensorflow/models/issues/4070#issuecomment-421201459

> Yes we initialize our backbone net weights from imagenet trained checkpoints.

So I must try it again

**Comment by BraginIvan** [comment]
Training MobilenetV1 from pre-trained backbone gives the same result (mAP 0.15).
Can you please provide details how you achieve such results?
Augmentation/Knowledge distillation/huge batch size/something else?

**Comment by BraginIvan** [comment]
Found config. batch size 1024, it is clear now /home/ivan/PycharmProjects/models/research/object_detection/samples/configs/ssd_mobilenet_v1_300x300_coco14_sync.config

**Comment by Hrayo712** [comment]
Hello @BraginIvan 

I hope you're doing well. I am trying to do a similar thing, so I was wondering if you managed to reproduce the results you were looking for, and if so, with which configuration settings.

**Comment by BraginIvan** [comment]
This works.
https://github.com/tensorflow/models/blob/master/research/object_detection/samples/configs/ssd_mobilenet_v1_300x300_coco14_sync.config
I trained on Google TPU and it took ~5-8 hours. 

Also tried on local RTX 2080TI with smaller batchsize and learning rate and got almost the same result as on TPU but it took 6 days.

**Comment by Hrayo712** [comment]
@BraginIvan 

Thanks for the quick response. Did you tried with SSD mobilenetv2 as well ? by any chance

**Comment by BraginIvan** [comment]
No, I developed for specific device and MobileNet V1 was faster then MobileNet V2
